code/modules/food_and_drinks/food/foods/meat.dm

```diff
@@ -67,12 +68,14 @@
 	desc = "This doesn't look like meat, but your standards aren't <i>that</i> high to begin with."
 	list_reagents = list("nutriment" = 3, "vitamin" = 2, "blood" = 5)
 	filling_color = rgb(150, 0, 0)
+	icon = 'icons/obj/food/meat.dmi'
```

Comment:
It inherits this from ```/obj/item/reagent_containers/food/snacks/meat``` already, this is the case here and in other places, i suggest taking another look on all of them.

Reply:
I was unaware that setting it for meat would make it inherit all the way down the chain for icon. However, given how the other icon swaps work this would probably only work for soup; which I can in-fact change for soup. I don't think there's any others it would work for other than that due to how things are pathed. I can totally change it for those though, thought I had to path it this way! Woulda saved me a decent chunk of time haha.